### ISSUE TITLE
support video tag?

### ISSUE THREAD

**Issue by tenbit** [dimsemenov/PhotoSwipe#1539]
now the plugin support images,but does not support video tag?

**Comment by acwolff** [comment]
No, but you can display videos as I do in [my jAlbum  PhotoSwipe skin](https://jalbum.net/en/skins/skin/PhotoSwipe), see [here]( https://andrewolff.jalbum.net/TestVideos_PS/) for an example. See also the items array in source file :https://andrewolff.jalbum.net/TestVideos_PS/slidePage.js

**Comment by tenbit** [comment]
i am sorry ,it's too difficult to me.

**Comment by tenbit** [comment]
i found the way .use the slide of html

**Comment by acwolff** [comment]
Yes indeed, here is the item code of the 3th item in [this album](https://andrewolff.jalbum.net/TestVideos_PS/)

      {
        videosrc: 'slides/IMG_0877.mp4',
        vw: 1080,
        vh: 1920,    
        html: '<video controls poster="slides/IMG_0877.jpg" ><source src="slides/IMG_0877.mp4" type="video/mp4"></video>',
        title:  'iPhone video file' 
      },

**Comment by dimsemenov** [comment]
Video support is planned in v6 https://github.com/dimsemenov/PhotoSwipe/discussions/2170

(for v5 there is a plugin)
